util/dvsim/testplanner/class_defs.py

```diff
@@ -267,6 +267,11 @@ def display(self):
         for entry in self.entries:
             entry.display()
 
+    def has_entries(self):
+        '''Returns True if testplan has entries, else False.
+        '''
+        return (len(self.entries) != 0)
```

Comment:
Nit: stray parens. This could be `return len(self.entries) != 0`.

Also, I'm not sure this function is really necessary. Instead of `if my_testplan.has_entries()`, you could write `if my_testplan.entries` (since the empty list is falsy in Python). If anything, I think the latter is probably clearer.

Reply:
Done, thanks!